ONSdigital/ras-party
# Adding password reset email change fix

# What and why?
This fixes the case where a user changes email while having requested a forgot password link by deleting the password reset token in the database when the party email is changed

# How to test?
- deploy to environment
- follow the `forgot password` path for an account
- check the token is in the database
- change the user email and verify the change through the link provided 
- check the token has been removed from the database and the counter is reset
- 
# Jira
https://jira.ons.gov.uk/browse/RAS-840
